apps/wallet-mobile/src/features/Swap/SwapNavigator.tsx

```diff
@@ -62,9 +64,37 @@ export const SwapTabNavigator = () => {
           })}
           style={styles.tab}
         >
-          <Tab.Screen name="token-swap" component={StartSwapOrderScreen} />
+          <Tab.Screen
+            listeners={{
+              tabPress: (e) => {
+                // if the keyboard is open, the user needs to close first the keyboard
+                // then, press again the tab to change screen
+                // to avoid screen freezing
+                if (isKeyboardOpen) {
+                  Keyboard.dismiss()
+                  e.preventDefault()
+                }
+              },
+            }}
+            name="token-swap"
+            component={StartSwapOrderScreen}
+          />
 
-          <Tab.Screen name="orders" getComponent={() => ListOrders} />
+          <Tab.Screen
```

Comment:
Why do we use `component` in one and `getComponent` with a function that returns the component in the other one?

Reply:
getComponent is lazy, since the start order is the first render, the second tab is only loaded when clicked. 
this bug looks like some animation fighting problem